LLM/rule_book/logger.py

```
@@ -0,0 +1,45 @@
+        "timestamp": datetime.now(timezone.utc).strftime("%Y-%m-%dT%H:%M:%SZ"),
+        "query": query,
+        "chunks_count": len(chunks),
+        "chunks_sources": [f"{c.get('source', '')} {c.get('article', '')}".strip() for c in chunks],
+        "answer": answer,
+        "error": error,
+        "elapsed_ms": elapsed_ms,
+    }
+    try:
+        await _write_log(record)
+    except Exception:
+        pass  # 로그 실패가 서비스에 영향을 주지 않도록
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)